### ISSUE TITLE
Historical meteor releases are required. They do not need to be supported, but are required.

### ISSUE THREAD

**Issue by etianl** [MeteorDevelopment/meteor-client#3457]
### Describe the feature

A historical page of old meteor releases is required, but maybe with a disclaimer stating that they will not be supported. The suggestion of use Multiconnect is not always valid due to the fact that it can not connect to every minecraft version, and in particular it cannot connect to minecraft version 1.19.1 or 1.19.2 if enfore secure chat is enabled. This is a severe issue at the moment due to tens of thousands of servers still being on those two problem versions. 

TL:DR, we still need meteor for 1.19.2 at the very least due to multiconnect and/or viafabric not being able to connect to enforce secure chat = true servers

### Before submitting a suggestion

- [X] This feature doesn't already exist in the client. (I have checked every module and their settings on the **latest dev build**)

- [X] This wasn't already suggested. (I have searched suggestions on GitHub and read the FAQ)

**Comment by GreemDev** [comment]
clone the repository at the commit right before the commit that updates the MC version and compile it locally via the readme instructions

**Comment by I-Got-You-Dead2** [comment]
“Required” says who?

**Comment by etianl** [comment]
I am a dingus, I just found this page
https://github.com/MeteorDevelopment/meteor-client/tags
anyone can build any version for themselves it appears :D

**Comment by GreemDev** [comment]
that being said pre-1.19 versions probably won't compile as some of their dependencies aren't on the meteor maven server anymore

**Comment by RacoonDog** [comment]
> that being said pre-1.19 versions probably won't compile as some of their dependencies aren't on the meteor maven server anymore

yeah but the dependencies are themselves open source as well

**Comment by GreemDev** [comment]
> > that being said pre-1.19 versions probably won't compile as some of their dependencies aren't on the meteor maven server anymore
> 
> yeah but the dependencies are themselves open source as well

true however you'd also need to edit the build.gradle to use local dependencies like that

**Comment by ManInMyVan** [comment]
[https://maven.meteordev.org](https://maven.meteordev.org)
